bot/exts/backend/error_handler.py

```diff
@@ -85,6 +86,11 @@ async def on_command_error(self, ctx: Context, e: errors.CommandError) -> None:
                 await ctx.send(f"{e.original} Please wait for it to finish and try again later.")
             elif isinstance(e.original, InvalidInfractedUserError):
                 await ctx.send(f"Cannot infract that user. {e.original.reason}")
+            elif isinstance(e.original, Forbidden):
+                if await try_handle_forbidden(e.original, ctx.message):
+                    # Forbidden error was handled so return
+                    return
+                await self.handle_unexpected_error(ctx, e.original)
```

Comment:
As discussed in Discord.

```suggestion
            elif isinstance(e.original, Forbidden):
                try:
                    await try_handle_forbidden(e.original, ctx.message)
                except Forbidden:
                    await self.handle_unexpected_error(ctx, e.original)
```

Reply:
e4b2d5e